FirebaseCore/Sources/FIRApp.m

```diff
@@ -22,7 +22,7 @@
 #import <AppKit/AppKit.h>
 #endif
 
-#import "FirebaseCore/Sources/Public/FIRApp.h"
+#import "FirebaseCore/Sources/Public/FirebaseCore/FIRApp.h"
```

Comment:
Does this constitute a breaking change? Just making sure, because I suspect it's OK because of CocoaPods flattening behavior, but am not sure.

Reply:
It's my understanding that the CocoaPods flattening makes this non-breaking.  See the FirebaseCore.podspec change.

 I plan to validate with more extensive testing including using this Core with old Firebase 6.x deps ...